src/pages/validate.tsx

```diff
@@ -112,7 +112,7 @@ function ClientIdentifierValidator() {
               be authorized to do so itself) and validate it.
             </Typography>
             <Typography variant="body1">
-              This will also check, if the REST resource is set up correctly and
+              We will also check if the correct accept headers are set and if
               the remote Client Identifier matches the one you enter here.
```

Comment:
This will check if there is a client identifier document located at URL, if it is served with the correct content-type header as JSON, and that it was not served via a redirect (i.e., a 200 status code).

(per the validation rules executing)

Reply:
changed with https://github.com/inrupt/client-identifier-helper/pull/142/commits/3fb41bb8a766a34fb242ca10d37f2060b97e58e6